### PR TITLE
Added the Profile/README.md and pull_request_template.md files

### PR DESCRIPTION
**Pull request by atheo89**
Hello all, this pull request contains a proposal about structuring the GitHub organization opendatahub-io. By using a special functionality of GitHub (1) we can display an overview page where the visitor could navigate himself into the repositories easier.  

(1) .github repository : https://www.freecodecamp.org/news/how-to-use-the-dot-github-repository/

My proposal is trying to solve (in a way) the non-semantic listing of the repositories. Github's default listing of the repositories is by title or the last modified time,  etc which actually does not give any information regarding the structure (which is the core software, what is the module/library etc). As a consequence, this discourages the open-source community to get involved. So, the idea was to list and categorize the repositories according to the project's intended purpose, subject area, or other important qualities, give initial quick how-to's and help the community to easily jump into our projects.

Long story short, please, take the time to review it and propose alternatives or extra features. 

PS: On this PR I uploaded as well the latest pull_request_template.md that we are using because in this way it can be visible in all the repositories under the org. (except if every repository should have its PR_template file)

### CODE REVIEW

**Comment by accorvin** [comment]
I love this. /lgtm

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Merge criteria:
+<!--- This PR will be merged by any repository approver when it meets all the points in the checklist -->
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+
+- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
+- [ ] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message.
+- [ ] JIRA link(s):
+- [ ] The Jira story is acked.
+- [ ] Live build image:
```
Comment (LaVLaS) [inline PULL_REQUEST_TEMPLATE.md:19]: Can you remove this since it is specific to the downstream repos

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	 * [odh-dashboard](https://github.com/opendatahub-io/odh-dashboard): A dashboard for Open Data Hub components. Shows what's installed and what's available for installation, as well as, links to component UIs and links to component documentation.
+	 * [kfp-tekton](https://github.com/opendatahub-io/kfp-tekton): Project bringing Kubeflow Pipelines and Tekton together. The current code allows you run Kubeflow Pipelines with Tekton backend end to end.
+
+  - **Tools**
+  	 *  [kubeflow](https://github.com/opendatahub-io/kubeflow): This repository contains the code of kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
+  	 *  [manifests](https://github.com/opendatahub-io/manifests): The Kubeflow Manifests repository is organized under three (3) main directories, which include manifests for installing: `apps`,  `common`, `contrib`
+  	 *   [odh-images](https://github.com/opendatahub-io/odh-images):  This repository contains code for the various Open Data Hub images.
```
Comment (LaVLaS) [inline profile/README.md:33]: I would remove this line since it will probably be moved along with the other Tier 2 repos.

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ 	 * [opendatahub.io](https://github.com/opendatahub-io/opendatahub.io):  Contains the webpage source code 
+ 	 * [opendatahub-community](https://github.com/opendatahub-io/opendatahub-community): This is the starting point for joining and contributing to the Open Data Hub community.
+ 	 * [jupyterhub-odh](https://github.com/opendatahub-io/jupyterhub-odh): This repository contains an example of a customised deployment of JupyterHub for OpenShift, which uses the same OpenShift cluster as the deployment is running in, as the authentication provider.
+
+ - **Template Repositories**
+	 * [odh-template-sig](https://github.com/opendatahub-io/odh-template-sig) Repository template for Open Data Hub Special Interest Groups.
+- **ODH Tutorials**
+	* [Fraud-Detection-with-Business-Workflows-Tutorial](https://github.com/opendatahub-io/Fraud-Detection-with-Business-Workflows-Tutorial)
+	* [data-engineering-and-machine-learning-workshop](https://github.com/opendatahub-io/data-engineering-and-machine-learning-workshop)
+
```
Comment (LaVLaS) [inline profile/README.md:49]: You can delete these because they are no longer current and need to be archived

* Fraud-Detection-with-Business-Workflows-Tutorial
* data-engineering-and-machine-learning-workshop

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ 	 * [jupyterhub-odh](https://github.com/opendatahub-io/jupyterhub-odh): This repository contains an example of a customised deployment of JupyterHub for OpenShift, which uses the same OpenShift cluster as the deployment is running in, as the authentication provider.
+
+ - **Template Repositories**
+	 * [odh-template-sig](https://github.com/opendatahub-io/odh-template-sig) Repository template for Open Data Hub Special Interest Groups.
+- **ODH Tutorials**
+	* [Fraud-Detection-with-Business-Workflows-Tutorial](https://github.com/opendatahub-io/Fraud-Detection-with-Business-Workflows-Tutorial)
+	* [data-engineering-and-machine-learning-workshop](https://github.com/opendatahub-io/data-engineering-and-machine-learning-workshop)
+ 	 
+ - **Backup Repos** 
+	 * [website](https://github.com/opendatahub-io/website): Contains code for the old website
```
Comment (LaVLaS) [inline profile/README.md:51]: This line can be deleted.  I think this repo was only a proof of concept and never used

**Review by LaVLaS — CHANGES_REQUESTED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (LaVLaS) [reply PULL_REQUEST_TEMPLATE.md:19]: Specifically
```
- [ ] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message.
- [ ] JIRA link(s):
- [ ] The Jira story is acked.
- [ ] Live build image: 
```

**Review by lucferbux — APPROVED** [review]
/lgtm

**Comment by LaVLaS** [comment]
@atheo89 Should we copy the `ISSUE_TEMPLATE` folder in https://github.com/opendatahub-io/.github/pull/1 to this repo to make it the org default?

**Comment by atheo89** [comment]
@LaVLaS  I think is a good idea, I am going to add it
